### DISCUSSION TITLE
Can we keep the inspector open until i say so to close

### DISCUSSION THREAD

**Discussion by Naveena1321** [clientIO/joint#2787] [Q&A] [answered]
### Introduction

I have a requirement like need to keep the inspector open until I click on a button to close

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
Closing the inspector is done at the application level, not at the library level. The inspector closes automatically only after the cell model is removed.

You can also use the `live: false` option to allow the user to decide whether to save or discard the changes. See [OrgChart](
https://www.jointjs.com/demos/organizational-chart) example.
